addons/portal/static/src/chatter/frontend/portal_chatter_service.js

```diff
@@ -19,16 +19,30 @@ export class PortalChatterService {
 
     async createShadow(root) {
         const shadow = root.attachShadow({ mode: "open" });
-        const res = await getBundle("portal.assets_chatter_style");
-        for (const url of res.cssLibs) {
-            const link = document.createElement("link");
-            link.rel = "stylesheet";
-            link.href = url;
-            shadow.appendChild(link);
-            await new Promise((res, rej) => {
-                link.addEventListener("load", res);
-                link.addEventListener("error", rej);
-            });
+        try {
+            const res = await getBundle("portal.assets_chatter_style");
+            for (const url of res.cssLibs) {
+                const link = document.createElement("link");
+                link.rel = "stylesheet";
+                link.href = url;
+                shadow.appendChild(link);
+                await new Promise((res, rej) => {
+                    link.addEventListener("load", res);
+                    link.addEventListener("error", rej);
+                });
+            }
+        } catch (e) {
+            if (e instanceof AssetsLoadingError && e.cause instanceof TypeError) {
+                // an AssetsLoadingError caused by a TypeError means that the
+                // fetch request has been cancelled by the browser. It can occur
+                // when the user changes page, or navigate away from the website
+                // client action, so the iframe is unloaded. In this case, we
+                // don't care abour reporting the error, it is actually a normal
+                // situation.
+                return new Promise(() => {});
```

Comment:
:pick: Shouldn't we handle this case in the call site as well? I fear another error could occur when mounting the app otherwise.

Reply:
`createShadow()` is not awaited by the caller, thus we cannot rely on try/catch in the caller code.
We could put a `catch()` that silently does the same, but this reduces Locality of Concern as the catch is about assets loading and from caller Point of View of `createShadow()` this is strange to manage `AssetsLoadingError` for a `createShadow()` function.